src/main/resources/db/migration/V1__create_initial_tables.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE food_chains (
+    id SERIAL PRIMARY KEY,
+     name VARCHAR NOT NULL,
+    description VARCHAR(255),
+    img_url TEXT NOT NULL
+);
+
+CREATE TABLE reviews (
+    id SERIAL PRIMARY KEY,
+    comment VARCHAR NOT NULL,
+    rating  INTEGER CHECK (rating <= '1' >= '5') NOT NULL,
+    reviews INTEGER REFERENCES food_chains(id)
```

Comment:
You want this to be 
`food_chain_id INTEGER REFERENCES food_chains(id)`

That way your reviews table has the `food_chain_id` column.